play-2.4/swagger-play2/build.sbt

```diff
@@ -6,6 +6,8 @@ checksums in update := Nil
 scalaVersion:= "2.11.6"
 crossScalaVersions := Seq("2.11.6", "2.11.7")
 
+resolvers += "scalaz-bintray" at "https://dl.bintray.com/scalaz/releases"
```

Comment:
I added this because when I cloned and compiled I got a resolution error (below). I checked Maven Central and couldn't find the artifact version `0.7a` on there (see http://mvnrepository.com/artifact/org.scalaz.stream/scalaz-stream_2.11)

```
[info] Resolving org.scalaz.stream#scalaz-stream_2.11;0.7a ...
[warn] 	module not found: org.scalaz.stream#scalaz-stream_2.11;0.7a
[warn] ==== local: tried
[warn]   /Users/Joseph/.ivy2/local/org.scalaz.stream/scalaz-stream_2.11/0.7a/ivys/ivy.xml
[warn] ==== activator-launcher-local: tried
[warn]   /Users/Joseph/.activator/repository/org.scalaz.stream/scalaz-stream_2.11/0.7a/ivys/ivy.xml
[warn] ==== activator-local: tried
[warn]   /opt/typesafe/activator-1.3.6/repository/org.scalaz.stream/scalaz-stream_2.11/0.7a/ivys/ivy.xml
[warn] ==== public: tried
[warn]   https://repo1.maven.org/maven2/org/scalaz/stream/scalaz-stream_2.11/0.7a/scalaz-stream_2.11-0.7a.pom
[warn] ==== typesafe-releases: tried
[warn]   http://repo.typesafe.com/typesafe/releases/org/scalaz/stream/scalaz-stream_2.11/0.7a/scalaz-stream_2.11-0.7a.pom
[warn] ==== typesafe-ivy-releasez: tried
[warn]   http://repo.typesafe.com/typesafe/ivy-releases/org.scalaz.stream/scalaz-stream_2.11/0.7a/ivys/ivy.xml
[info] Resolving jline#jline;2.12.1 ...
[info] downloading https://repo1.maven.org/maven2/org/yaml/snakeyaml/1.12/snakeyaml-1.12.jar ...
[info] 	[SUCCESSFUL ] org.yaml#snakeyaml;1.12!snakeyaml.jar(bundle) (156ms)
[info] downloading https://repo1.maven.org/maven2/com/google/code/findbugs/jsr305/2.0.1/jsr305-2.0.1.jar ...
[info] 	[SUCCESSFUL ] com.google.code.findbugs#jsr305;2.0.1!jsr305.jar (22ms)
[info] downloading https://repo1.maven.org/maven2/com/typesafe/play/play-ebean_2.11/2.0.0/play-ebean_2.11-2.0.0.jar ...
[info] 	[SUCCESSFUL ] com.typesafe.play#play-ebean_2.11;2.0.0!play-ebean_2.11.jar (18ms)
[info] downloading https://repo1.maven.org/maven2/org/specs2/specs2-core_2.11/3.6/specs2-core_2.11-3.6.jar ...
[info] 	[SUCCESSFUL ] org.specs2#specs2-core_2.11;3.6!specs2-core_2.11.jar (615ms)
[info] downloading https://repo1.maven.org/maven2/org/specs2/specs2-mock_2.11/3.6/specs2-mock_2.11-3.6.jar ...
[info] 	[SUCCESSFUL ] org.specs2#specs2-mock_2.11;3.6!specs2-mock_2.11.jar (77ms)
[info] downloading https://repo1.maven.org/maven2/org/specs2/specs2-junit_2.11/3.6/specs2-junit_2.11-3.6.jar ...
[info] 	[SUCCESSFUL ] org.specs2#specs2-junit_2.11;3.6!specs2-junit_2.11.jar (91ms)
[info] downloading https://repo1.maven.org/maven2/org/mockito/mockito-core/1.9.5/mockito-core-1.9.5.jar ...
[info] 	[SUCCESSFUL ] org.mockito#mockito-core;1.9.5!mockito-core.jar (426ms)
[info] downloading https://repo1.maven.org/maven2/com/typesafe/play/play-jdbc-evolutions_2.11/2.4.2/play-jdbc-evolutions_2.11-2.4.2.jar ...
[info] 	[SUCCESSFUL ] com.typesafe.play#play-jdbc-evolutions_2.11;2.4.2!play-jdbc-evolutions_2.11.jar (80ms)
[info] downloading https://repo1.maven.org/maven2/org/avaje/ebeanorm/avaje-ebeanorm/6.8.1/avaje-ebeanorm-6.8.1.jar ...
[info] 	[SUCCESSFUL ] org.avaje.ebeanorm#avaje-ebeanorm;6.8.1!avaje-ebeanorm.jar (531ms)
[info] downloading https://repo1.maven.org/maven2/org/avaje/ebeanorm/avaje-ebeanorm-agent/4.7.1/avaje-ebeanorm-agent-4.7.1.jar ...
[info] 	[SUCCESSFUL ] org.avaje.ebeanorm#avaje-ebeanorm-agent;4.7.1!avaje-ebeanorm-agent.jar (93ms)
[info] downloading https://repo1.maven.org/maven2/org/jetbrains/annotations/13.0/annotations-13.0.jar ...
[info] 	[SUCCESSFUL ] org.jetbrains#annotations;13.0!annotations.jar (13ms)
[info] downloading https://repo1.maven.org/maven2/javax/persistence/persistence-api/1.0/persistence-api-1.0.jar ...
[info] 	[SUCCESSFUL ] javax.persistence#persistence-api;1.0!persistence-api.jar (38ms)
[info] downloading https://repo1.maven.org/maven2/org/specs2/specs2-matcher_2.11/3.6/specs2-matcher_2.11-3.6.jar ...
[info] 	[SUCCESSFUL ] org.specs2#specs2-matcher_2.11;3.6!specs2-matcher_2.11.jar (912ms)
[info] downloading https://repo1.maven.org/maven2/org/specs2/specs2-common_2.11/3.6/specs2-common_2.11-3.6.jar ...
[info] 	[SUCCESSFUL ] org.specs2#specs2-common_2.11;3.6!specs2-common_2.11.jar (661ms)
[info] downloading https://repo1.maven.org/maven2/org/scalaz/scalaz-core_2.11/7.1.1/scalaz-core_2.11-7.1.1.jar ...
[info] 	[SUCCESSFUL ] org.scalaz#scalaz-core_2.11;7.1.1!scalaz-core_2.11.jar(bundle) (2485ms)
[info] downloading https://repo1.maven.org/maven2/org/scalaz/scalaz-concurrent_2.11/7.1.1/scalaz-concurrent_2.11-7.1.1.jar ...
[info] 	[SUCCESSFUL ] org.scalaz#scalaz-concurrent_2.11;7.1.1!scalaz-concurrent_2.11.jar(bundle) (124ms)
[info] downloading https://repo1.maven.org/maven2/org/scalaz/scalaz-effect_2.11/7.1.1/scalaz-effect_2.11-7.1.1.jar ...
[info] 	[SUCCESSFUL ] org.scalaz#scalaz-effect_2.11;7.1.1!scalaz-effect_2.11.jar(bundle) (127ms)
[info] downloading https://repo1.maven.org/maven2/org/objenesis/objenesis/1.0/objenesis-1.0.jar ...
[info] 	[SUCCESSFUL ] org.objenesis#objenesis;1.0!objenesis.jar (20ms)
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	::          UNRESOLVED DEPENDENCIES         ::
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	:: org.scalaz.stream#scalaz-stream_2.11;0.7a: not found
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 
[warn] 	Note: Unresolved dependencies path:
[warn] 		org.scalaz.stream:scalaz-stream_2.11:0.7a
[warn] 		  +- org.specs2:specs2-common_2.11:3.6
[warn] 		  +- org.specs2:specs2-matcher_2.11:3.6
[warn] 		  +- org.specs2:specs2-core_2.11:3.6 (/Users/Joseph/Workspace/swagger-play/play-2.4/swagger-play2/build.sbt#L9-18)
[warn] 		  +- org.specs2:specs2-junit_2.11:3.6 (/Users/Joseph/Workspace/swagger-play/play-2.4/swagger-play2/build.sbt#L9-18)
[warn] 		  +- io.swagger:swagger-play2_2.11:1.5.0-SNAPSHOT
sbt.ResolveException: unresolved dependency: org.scalaz.stream#scalaz-stream_2.11;0.7a: not found
	at sbt.IvyActions$.sbt$IvyActions$$resolve(IvyActions.scala:291)
	at sbt.IvyActions$$anonfun$updateEither$1.apply(IvyActions.scala:188)
	at sbt.IvyActions$$anonfun$updateEither$1.apply(IvyActions.scala:165)
	at sbt.IvySbt$Module$$anonfun$withModule$1.apply(Ivy.scala:155)
	at sbt.IvySbt$Module$$anonfun$withModule$1.apply(Ivy.scala:155)
	at sbt.IvySbt$$anonfun$withIvy$1.apply(Ivy.scala:132)
	at sbt.IvySbt.sbt$IvySbt$$action$1(Ivy.scala:57)
	at sbt.IvySbt$$anon$4.call(Ivy.scala:65)
	at xsbt.boot.Locks$GlobalLock.withChannel$1(Locks.scala:93)
	at xsbt.boot.Locks$GlobalLock.xsbt$boot$Locks$GlobalLock$$withChannelRetries$1(Locks.scala:78)
	at xsbt.boot.Locks$GlobalLock$$anonfun$withFileLock$1.apply(Locks.scala:97)
	at xsbt.boot.Using$.withResource(Using.scala:10)
	at xsbt.boot.Using$.apply(Using.scala:9)
	at xsbt.boot.Locks$GlobalLock.ignoringDeadlockAvoided(Locks.scala:58)
	at xsbt.boot.Locks$GlobalLock.withLock(Locks.scala:48)
	at xsbt.boot.Locks$.apply0(Locks.scala:31)
	at xsbt.boot.Locks$.apply(Locks.scala:28)
	at sbt.IvySbt.withDefaultLogger(Ivy.scala:65)
	at sbt.IvySbt.withIvy(Ivy.scala:127)
	at sbt.IvySbt.withIvy(Ivy.scala:124)
	at sbt.IvySbt$Module.withModule(Ivy.scala:155)
	at sbt.IvyActions$.updateEither(IvyActions.scala:165)
	at sbt.Classpaths$$anonfun$sbt$Classpaths$$work$1$1.apply(Defaults.scala:1369)
	at sbt.Classpaths$$anonfun$sbt$Classpaths$$work$1$1.apply(Defaults.scala:1365)
	at sbt.Classpaths$$anonfun$doWork$1$1$$anonfun$87.apply(Defaults.scala:1399)
	at sbt.Classpaths$$anonfun$doWork$1$1$$anonfun$87.apply(Defaults.scala:1397)
	at sbt.Tracked$$anonfun$lastOutput$1.apply(Tracked.scala:37)
	at sbt.Classpaths$$anonfun$doWork$1$1.apply(Defaults.scala:1402)
	at sbt.Classpaths$$anonfun$doWork$1$1.apply(Defaults.scala:1396)
	at sbt.Tracked$$anonfun$inputChanged$1.apply(Tracked.scala:60)
	at sbt.Classpaths$.cachedUpdate(Defaults.scala:1419)
	at sbt.Classpaths$$anonfun$updateTask$1.apply(Defaults.scala:1348)
	at sbt.Classpaths$$anonfun$updateTask$1.apply(Defaults.scala:1310)
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:40)
	at sbt.std.Transform$$anon$4.work(System.scala:63)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:226)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:226)
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
	at sbt.Execute.work(Execute.scala:235)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:226)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:226)
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:159)
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
```